docker-compose.yml

```diff
@@ -105,8 +105,8 @@ services:
 
   kafkacat:
     image: confluentinc/cp-kafkacat
-    command: 
-      - bash 
+    command:
```

Comment:
Please remove the whole file modification, not concerning the feature

Reply:
Rollback changes